app/src/main/java/cm/aptoide/pt/view/DeepLinkManager.java

```diff
@@ -232,7 +269,13 @@ private void downloadNotificationDeepLink() {
   private void fromTimelineDeepLink(Intent intent) {
     deepLinkAnalytics.timelineNotification();
     String cardId = intent.getStringExtra(DeepLinkIntentReceiver.DeepLinksKeys.CARD_ID);
-    tabNavigator.navigate(new AppsTimelineTabNavigation(cardId));
+    if (intent.hasExtra(FROM_SHORTCUT)) {
+      timelineAnalytics.sendTimelineTabOpenedFromShortcut();
+      appShortcutsAnalytics.shortcutNavigation(ShortcutDestinations.TIMELINE);
+      tabNavigator.navigate(new AppsTimelineTabNavigation(cardId));
+    } else {
+      tabNavigator.navigate(new AppsTimelineTabNavigation(cardId));
+    }
```

Comment:
No need for the if else. "tabNavigator.navigate(new AppsTimelineTabNavigation(cardId));" it can be after the if statement

Reply:
Fixed